src/types/user/userDetails.ts

```diff
@@ -0,0 +1,16 @@
+import { Moment } from 'moment';
+
+export default interface UserDetails {
+  id?: string;
+  passsword?: string;
+  name: string;
+  email: string;
+  type: string;
+  joiningDate: Moment | null;
+  description: string;
+  phone: string;
+  isActive: boolean;
+  profileImage: string;
+  createdAt: Date;
+  label: string;
+}
```

Comment:
@aritrasen12345 Is this interface relevant to this project? do we have `joiningDate` at all?